desktop_version/src/Screen.cpp

```diff
@@ -170,9 +170,9 @@ void Screen::ResizeScreen(int x, int y)
     else
     {
         int result = SDL_SetWindowFullscreen(m_window, 0);
-        recacheTextures();
         if (result != 0)
         {
+            recacheTextures();
```

Comment:
nit: We could probably remove the `return;` and instead make the `if` just below it an `else if`, so that we effectively move recacheTextures to the bottom and avoid any lurking early return bugs.